compat/src/main/scala-2.11_2.12/scala/util/package.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Scala (https://www.scala-lang.org)
+ *
+ * Copyright EPFL and Lightbend, Inc.
+ *
+ * Licensed under Apache License 2.0
+ * (http://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package scala
+
+package object util {
```

Comment:
Does it have to be a `package object`? Could we put the `object chaining` definition in the package `scala.util` instead?

Reply:
that's an interesting question. I just copied from the stdlib. what are the practical differences between the two?

Reply:
putting class and object definitions into package objects is an anti-pattern (mostly because bugs are likely), that's an oversight in the std lib

Reply:
ok, I'll move it out of the package object.

is it binary compatible if we do that in the stdlib as well?